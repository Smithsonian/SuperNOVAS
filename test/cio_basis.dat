
2450545.000000 TEST       S2 O0 A1:     1.000000     0.000005     0.000268    -0.000005     1.000000     0.000042    -0.000268    -0.000042     1.000000 

2450545.000000 TEST       S2 O1 A1:     1.000000     0.000000     0.000268    -0.000000     1.000000     0.000042    -0.000268    -0.000042     1.000000 

2450545.000000 TEST       S2 O2 A1:     1.000000     0.000005     0.000268    -0.000005     1.000000     0.000042    -0.000268    -0.000042     1.000000 

2450545.000000 SUN        S0 O0 A1:     1.000000     0.000005     0.000268    -0.000005     1.000000     0.000042    -0.000268    -0.000042     1.000000 

2450545.000000 SUN        S0 O1 A1:     1.000000     0.000005     0.000268    -0.000005     1.000000     0.000042    -0.000268    -0.000042     1.000000 

2450545.000000 SUN        S0 O2 A1:     1.000000     0.000005     0.000268    -0.000005     1.000000     0.000042    -0.000268    -0.000042     1.000000 

2451545.000000 TEST       S2 O0 A1:     1.000000     0.000062     0.000027    -0.000062     1.000000     0.000028    -0.000027    -0.000028     1.000000 

2451545.000000 TEST       S2 O1 A1:     1.000000     0.000062     0.000027    -0.000062     1.000000     0.000028    -0.000027    -0.000028     1.000000 

2451545.000000 TEST       S2 O2 A1:     1.000000     0.000062     0.000027    -0.000062     1.000000     0.000028    -0.000027    -0.000028     1.000000 

2451545.000000 SUN        S0 O0 A1:     1.000000     0.000062     0.000027    -0.000062     1.000000     0.000028    -0.000027    -0.000028     1.000000 

2451545.000000 SUN        S0 O1 A1:     1.000000     0.000062     0.000027    -0.000062     1.000000     0.000028    -0.000027    -0.000028     1.000000 

2451545.000000 SUN        S0 O2 A1:     1.000000     0.000062     0.000027    -0.000062     1.000000     0.000028    -0.000027    -0.000028     1.000000 

2452545.000000 TEST       S2 O0 A1:     1.000000     0.000075    -0.000233    -0.000075     1.000000    -0.000016     0.000233     0.000016     1.000000 

2452545.000000 TEST       S2 O1 A1:     1.000000     0.000000    -0.000233    -0.000000     1.000000    -0.000016     0.000233     0.000016     1.000000 

2452545.000000 TEST       S2 O2 A1:     1.000000     0.000075    -0.000233    -0.000075     1.000000    -0.000016     0.000233     0.000016     1.000000 

2452545.000000 SUN        S0 O0 A1:     1.000000     0.000075    -0.000233    -0.000075     1.000000    -0.000016     0.000233     0.000016     1.000000 

2452545.000000 SUN        S0 O1 A1:     1.000000     0.000075    -0.000233    -0.000075     1.000000    -0.000016     0.000233     0.000016     1.000000 

2452545.000000 SUN        S0 O2 A1:     1.000000     0.000075    -0.000233    -0.000075     1.000000    -0.000016     0.000233     0.000016     1.000000 
