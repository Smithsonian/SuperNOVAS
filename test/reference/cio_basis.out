-10000.0 A0:  0.999996545  0.000000127  0.002628773 -0.000000096  1.000000000 -0.000011744 -0.002628773  0.000011744  0.999996545 
     0.0 A0:  0.999999998 -0.000061819 -0.000000081  0.000061819  0.999999998 -0.000000033  0.000000081  0.000000033  1.000000000 
 10000.0 A0:  0.999996407  0.000000133 -0.002680797 -0.000000192  1.000000000 -0.000022249  0.002680797  0.000022249  0.999996406 
-10000.0 A1:  0.999997  0.000000  0.002629 -0.000000  1.000000 -0.000012 -0.002629  0.000012  0.999997 
     0.0 A1:  1.000000 -0.000062 -0.000000  0.000062  1.000000 -0.000000  0.000000  0.000000  1.000000 
 10000.0 A1:  0.999996  0.000000 -0.002681 -0.000000  1.000000 -0.000022  0.002681  0.000022  0.999996 
